workshop/buildspec-node.yaml

```diff
@@ -0,0 +1,46 @@
+version: 0.2
+
+env:
+    variables:
+      IMAGE_TAG : "v1"
```

Comment:
@YogeshBhatiaAWS 

Do you intend to use the same versioning convention ? If yes, i would assume you want to have this be consistent with how the java app is versioned.

Reply:
image tag is anyway coming from environment variable

Reply:
Yeah, but to anyone new helping this will be the convention. Let's use consistent across the board.